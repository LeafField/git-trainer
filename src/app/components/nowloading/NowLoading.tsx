import React, { FC } from "react";

const NowLoading: FC = () => {
  return (
    <div
      className="w-fit animate-pulse text-3xl font-bold text-white"
      data-testid="nowloading"
    >
      ...NowLoading
    </div>
  );
};

export default NowLoading;
