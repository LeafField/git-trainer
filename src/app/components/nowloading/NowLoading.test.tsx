import { screen, render } from "@testing-library/react";
import NowLoading from "./NowLoading";

describe("NowLoadingのレンダリングテスト", () => {
  test("NowLoadingが正しくレンダリングされているか", () => {
    const { container } = render(<NowLoading />);
    expect(screen.getByTestId("nowloading")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
