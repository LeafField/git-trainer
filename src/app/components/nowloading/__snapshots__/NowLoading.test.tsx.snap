// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NowLoadingのレンダリングテスト NowLoadingが正しくレンダリングされているか 1`] = `
<div>
  <div
    class="w-fit animate-pulse text-3xl font-bold text-white"
    data-testid="nowloading"
  >
    ...NowLoading
  </div>
</div>
`;
